src/main/resources/static/data-upload.js

```diff
@@ -83,39 +89,43 @@ function uploadFile() {
         }, 3000);
 
         if (isMappingSelected) {
-            //TODO: resolve uploading-mapping issue
-            // var upload_mapping_data = new FormData($("#upload-mapping-form")[0]); console.log(upload_mapping_data);
-            // $.ajax({
-            //     url: baseApp+"/upload/mapping",
-            //     type: "POST",
-            //     data: upload_mapping_data,
-            //     processData: false, // Don't process the files
-            //     // contentType: false,
-            //     cache: false,
-            //     dataType: 'json',
-            //     success: function (feedback) {
-            //         // Handle upload success
-            //         var info = '<span id="mapping-alert" class="alert alert-success">Mapping succesfully uploaded</span>';
-            //         $("#message-board").append(info);
-            //         isMappingUploaded = true;
-            //         if (!isDirected && isDataUploaded) {
-            //             // window.location.replace("/mapping");
-            //             console.log(feedback);
-            //             isDirected = true;
-            //         }
-            //     },
-            //     error: function () {
-            //         // Handle upload error
-            //         var info = '<span id="mapping-alert" class="alert alert-danger">Mapping not uploaded</span>';
-            //         $("#message-board").append(info);
-            //         isMappingUploaded = false;
-            //     }
-            // });
-            // window.setTimeout(function () {
-            //     $("#mapping-alert").fadeTo(500, 0).slideUp(500, function () {
-            //         $(this).empty();
-            //     });
-            // }, 3000);
+            var upload_mapping_data = new FormData($("#upload-mapping-form")[0]);
+            $.ajax({
+                headers: {
+                    'Accept': 'application/json',
+                    'Content-Type': 'application/json'
+                },
+                url: baseApp+"/upload/mapping",
```

Comment:
This route i supposed for uploading JSON by mapping view. Not for uploading user file. 
For now we just disable upload-mapping file in the data view - we can enable it when we implement it.
Besides isn't it a copy paste from the mapping.js?